README.md

```diff
@@ -1,9 +1,9 @@
 .NET Core Documentation
```

Comment:
I'm wondering if we want to change this topic to be more broad since we cover .NET Standard and Languages (C# and F#) too. /cc @BillWagner @cartermp 
Might be out of scope for this PR though.